recipes/txtorcon/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "txtorcon" %}
+{% set version = "0.19.3" %}
+{% set sha256 = "f73396667909a3c7a98f4dd865edf4ed6a2518ee5a935d92e18b8a479ec244fd" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - twisted >=15.5.0
+    - ipaddress >=1.0.16  # [py2k]
+    - zope.interface >=3.6.1
+    - incremental
+    - automat
+
+test:
+  requires:
+    - pyopenssl
```

Comment:
Twisted comes from the `defaults` channel and isn't provided by conda-forge. I would think that `pyopenssl` should be a run requirement. If you look at the `txtorcon` requirements file, it actually says that you need the tls version of twisted. That pulls in a few extra things including `pyopenssl`:

https://github.com/twisted/twisted/blob/trunk/src/twisted/python/_setup.py#L92-L97

I'm not sure if we need the other listed packages.